### ISSUE TITLE
[docs] Admob testing section spelling error

### ISSUE THREAD

**Issue by julste** [expo/expo#11577]
I just copied the testing section from admob [page](https://docs.expo.io/versions/latest/sdk/admob/) to my code:

```
import Constants from 'expo-constants';

const testID = 'google-test-id';
const productionID = 'my-id';
// Is a real device and running in production.
const adUnitID = Constants.isDevice && !__DEV__ ? productionId : testID;
```

The const productionID is misspelled in the const adUnitID. Needs to be a capital D.

**Comment by AdamJNavarro** [comment]
Hey @julste, thanks for catching this! We would very much welcome a PR from you if this is something you're comfortable with. We have a [guide](https://github.com/expo/expo/blob/master/CONTRIBUTING.md#-updating-documentation) on contributing to our documentation. The important thing here would be to make sure that you apply the changes to all the applicable sdk versions of the documentation. Let me know if you have any questions or concerns!

Cheers,
Adam

**Comment by Simek** [comment]
The issue has been fixed in #12189.
